docs/configuration.txt

```diff
@@ -1125,6 +1125,39 @@ Metadata
   Defines how often the cluster collect metadata information
   (e.g. disk usages etc.) if no concrete  event is triggered.
 
+Gateway Settings
+----------------
+  The gateway persists cluster meta data on disk every time the meta data
+  changes. This data is stored persistently across full cluster restarts and
+  recovered after nodes are started again.
+
+.. _gateway.expected_nodes:
+
+**gateway.expected_nodes**
+  | *Default:*   ``-1``
+  | *Runtime:*  ``no``
+
+  The setting ``gateway.expected_nodes`` defines the number of nodes should be
+  waited for until the cluster state is recovered immediately.
+
+.. _gateway.recover_after_time:
+
+**gateway.recover_after_time**
+  | *Default:*   ``5m``
+  | *Runtime:*  ``no``
+
+  The ``gateway.recover_after_time`` setting defines the time to wait before
```

Comment:
the descriptions in https://crate.io/docs/en/latest/best_practice/multi_node_setup.html#gateway-configuration make much more sense to me.